extensions/context/examples/example.json

```diff
@@ -1,9 +1,55 @@
 {
   "type": "FeatureCollection",
-  "features": [ ],
   "context": {
-    "limit": 10,
-    "matched": 1092873,
-    "returned": 9
-  }
+    "returned": 1,
```

Comment:
this example is not correct, since the "returned" value matches the number of features.